src/hotspot/os/posix/signals_posix.cpp

```diff
@@ -1648,3 +1637,30 @@ void PosixSignals::do_resume(OSThread* osthread) {
 
   guarantee(osthread->sr.is_running(), "Must be running!");
 }
+
+void PosixSignals::do_task(Thread* thread, os::SuspendedThreadTask* task) {
+  if (PosixSignals::do_suspend(thread->osthread())) {
```

Comment:
Shouldn't need PosixSignals:: prefix in this method.

Reply:
Leftover from the time when do_task wasn't in signals_posix.cpp, fixed.